MdePkg/Include/IndustryStandard/SerialPortConsoleRedirectionTable.h

```diff
@@ -164,6 +164,7 @@ typedef struct {
 //
 // Baud Rate
 //
+#define EFI_ACPI_SERIAL_PORT_CONSOLE_REDIRECTION_TABLE_BAUD_RATE_AS_IS   0

```

Comment:
Please put the motivation and the spec references from the PR description into this commit log. (The PR description is not recorded into Git)